packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/ExtractionService.java

```diff
@@ -201,7 +202,26 @@ public Future<DocumentAsset> extractPDF(
 					while (entry != null) {
 						log.info("Adding {} to filemap", entry.getName());
 						final String filenameNoExt = removeFileExtension(entry.getName());
-						fileMap.put(filenameNoExt, zipEntryToBytes(zipInputStream));
+						final byte[] bytes = zipEntryToBytes(zipInputStream);
+
+						fileMap.put(filenameNoExt, bytes);
+						if (entry != null && entry.getName().toLowerCase().endsWith(".json")) {
+							ObjectMapper objectMapper = new ObjectMapper();
+
+							JsonNode rootNode = objectMapper.readTree(bytes);
+							if (rootNode.isArray()) {
+								ArrayNode arrayNode = (ArrayNode) rootNode;
+								String target = "Abstract";
```

Comment:
As a java formatting convention, these sort of definitions are usually declared as static constants at the class level rather than inline like this.  Something like:

```
private static final String TARGET = "Abstract";
private static final String PARAM = "detect_cls";
```